src/amo/components/SearchPage.js

```diff
@@ -45,9 +46,12 @@ export default class SearchPage extends React.Component {
 
     return (
       <div className="SearchPage">
+        {hasSearchParams ? (
```

Comment:
I feel like it's weird to show it with no params (and arguably whenever there are no results, actually).

Hitting that search page without a query gives a "please enter a search term" message. Adding bits that only appear once there are results to filter seems like bad UI/UX to me.